### PR TITLE
fix(enodebd): Fixing clear_stats function traversal of stats dictionary

### PR DESCRIPTION
**Pull request by ardzoht**
Signed-off-by: Alex Rodriguez <ardzoht@gmail.com>

## Summary

- testing on enodebd, there is an issue raised out on clearing of metrics due to missed traversal of dictionary items:
```
Jun 29 18:30:10 handset-test enodebd[1963471]: INFO:root:Clearing performance counter statistics
Jun 29 18:30:10 handset-test enodebd[1963471]: ERROR:asyncio:Exception in callback StatsManager._periodic_check_rf_tx()
Jun 29 18:30:10 handset-test enodebd[1963471]: handle: <TimerHandle when=527272.986716413 StatsManager._periodic_check_rf_tx()>
Jun 29 18:30:10 handset-test enodebd[1963471]: Traceback (most recent call last):
Jun 29 18:30:10 handset-test enodebd[1963471]:   File "/usr/lib/python3.8/asyncio/events.py", line 81, in _run
Jun 29 18:30:10 handset-test enodebd[1963471]:     self._context.run(self._callback, *self._args)
Jun 29 18:30:10 handset-test enodebd[1963471]:   File "/usr/local/lib/python3.8/dist-packages/magma/enodebd/stats_manager.py", line 94, in _periodic_check_rf_tx
Jun 29 18:30:10 handset-test enodebd[1963471]:     self._check_rf_tx()
Jun 29 18:30:10 handset-test enodebd[1963471]:   File "/usr/local/lib/python3.8/dist-packages/magma/enodebd/stats_manager.py", line 116, in _check_rf_tx
Jun 29 18:30:10 handset-test enodebd[1963471]:     self._check_rf_tx_for_handler(handler)
Jun 29 18:30:10 handset-test enodebd[1963471]:   File "/usr/local/lib/python3.8/dist-packages/magma/enodebd/stats_manager.py", line 121, in _check_rf_tx_for_handler
Jun 29 18:30:10 handset-test enodebd[1963471]:     self._clear_stats()
Jun 29 18:30:10 handset-test enodebd[1963471]:   File "/usr/local/lib/python3.8/dist-packages/magma/enodebd/stats_manager.py", line 366, in _clear_stats
Jun 29 18:30:10 handset-test enodebd[1963471]:     for pm_name, metric in self.PM_FILE_TO_METRIC_MAP:
Jun 29 18:30:10 handset-test enodebd[1963471]: ValueError: too many values to unpack (expected 2)
```

## Test Plan

- added unit test to cover edge case
- make test on local VM

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/enodebd/stats_manager.py [hunk 1]
```diff
@@ -370,7 +370,7 @@ def _clear_stats(self) -> None:
         """
         logger.info('Clearing performance counter statistics')
         # Set all metrics to 0 if eNodeB not connected
-        for pm_name, metric in self.PM_FILE_TO_METRIC_MAP:
+        for pm_name, metric in self.PM_FILE_TO_METRIC_MAP.items():
```
Comment (pshelar) [inline lte/gateway/python/magma/enodebd/stats_manager.py:373]: any of the stats unit tests covers this ?

Comment (ardzoht) [reply lte/gateway/python/magma/enodebd/stats_manager.py:373]: added an unit test that covers it

**Review by pshelar — APPROVED** [review]
LGTM
